Mac/ErrorHandler.swift

```diff
@@ -10,6 +10,11 @@ import AppKit
 import Account
 import os.log
 
+// asserts that OSLog is a sendable type
+// @preconcurrency import os.log _should_ resolve the warning in this scenario, but does
+// not due to a bug (in Swift 5.10)
+extension OSLog: @unchecked Sendable { }
```

Comment:
In this specific case, yes - but only because I was reading about it on the Swift forums (see [this tidbit](https://forums.swift.org/t/swift-5-10-concurrency-and-xctest/69929/15) - which references [this from Quinn in the developer forums](https://forums.developer.apple.com/forums/thread/747816?answerId=781922022#781922022)). The ideal solution here would be to abrogate the warnings with `@preconcurrency import os.log`, but that's apparently not available right now.

I will add the bit that Quinn suggested though, just to make sure:
```
#if swift(>=6.0)
    #warning("Reevaluate whether this decoration is necessary.")
#endif
```

Reply:
If you'd prefer to play it safe, I can absolutely pull this all and leave it alone.